third_party/terraform/resources/resource_cloudfunctions_function.go

```diff
@@ -179,7 +171,7 @@ func resourceCloudFunctionsFunction() *schema.Resource {
 			"available_memory_mb": {
 				Type:        schema.TypeInt,
 				Optional:    true,
-				Default:     functionDefaultAllowedMemoryMb,
+				Default:     256,
```

Comment:
Why move all these values?  It doesn't seem like this is a change from the previous behavior.

Reply:
I changed it the because those constants are doing much. Other resources don't do this